### PR TITLE
[improvement] Adding logging to LoC Service

### PR DESCRIPTION
**Pull request by ChrisCarini**
# Summary

Adding some debug logging to the `LoCService` and the `GitNumStat` class.

### CODE REVIEW

File: src/main/java/com/chriscarini/jetbrains/locchangecountdetector/LoCService.java [hunk 1]
```diff
@@ -70,10 +75,12 @@ public void after(@NotNull List<? extends VFileEvent> events) {
     }
 
     private Pair<Integer, Integer> getGitShowHeadNumStat() {
+        LOG.debug(String.format("%s - Running `git show --numstat`...", project.getName()));
```
Comment (bansalsulabh) [inline src/main/java/com/chriscarini/jetbrains/locchangecountdetector/LoCService.java:78]: Nice, would be easy to debug if shit happens!

**Review by bansalsulabh — DISMISSED** [review]
LGTM!!

File: src/main/java/com/chriscarini/jetbrains/locchangecountdetector/LoCService.java (hunk 1 above)
Comment (ChrisCarini) [reply src/main/java/com/chriscarini/jetbrains/locchangecountdetector/LoCService.java:78]: yeah, we'll have to turn on debug logging, but it's nice to have these! In future, we can add more logging to parts of the plugin too.

**Review by bansalsulabh — APPROVED** [review]
lgtm!!

**Pull request merged**